controllers/lagoonbuild_controller.go

```diff
@@ -46,29 +47,33 @@ import (
 // LagoonBuildReconciler reconciles a LagoonBuild object
 type LagoonBuildReconciler struct {
 	client.Client
-	Log                   logr.Logger
-	Scheme                *runtime.Scheme
-	EnableMQ              bool
-	Messaging             *handlers.Messaging
-	BuildImage            string
-	IsOpenshift           bool
-	NamespacePrefix       string
-	RandomNamespacePrefix bool
-	ControllerNamespace   string
-	EnableDebug           bool
-	FastlyServiceID       string
-	FastlyWatchStatus     bool
+	Log                     logr.Logger
+	Scheme                  *runtime.Scheme
+	EnableMQ                bool
+	Messaging               *handlers.Messaging
+	BuildImage              string
+	IsOpenshift             bool
+	NamespacePrefix         string
+	RandomNamespacePrefix   bool
+	ControllerNamespace     string
+	EnableDebug             bool
+	FastlyServiceID         string
+	FastlyWatchStatus       bool
 	// BuildPodRunAsUser sets the build pod securityContext.runAsUser value.
-	BuildPodRunAsUser int64
+	BuildPodRunAsUser       int64
 	// BuildPodRunAsGroup sets the build pod securityContext.runAsGroup value.
-	BuildPodRunAsGroup int64
+	BuildPodRunAsGroup      int64
 	// BuildPodFSGroup sets the build pod securityContext.fsGroup value.
-	BuildPodFSGroup int64
+	BuildPodFSGroup         int64
 	// Lagoon feature flags
-	LFFForceRootlessWorkload         string
-	LFFDefaultRootlessWorkload       string
-	LFFForceIsolationNetworkPolicy   string
-	LFFDefaultIsolationNetworkPolicy string
+	LFFForceRootlessWorkload            string
+	LFFDefaultRootlessWorkload          string
+	LFFForceIsolationNetworkPolicy      string
+	LFFDefaultIsolationNetworkPolicy    string
+	MonthlyBackupDefaultRetention       int
+	WeeklyBackupDefaultRetention        int
+	DailyBackupDefaultRetention         int
+	K8upWeeklyRandomFeatureFlag         bool
```

Comment:
Could the `*BackupDefaultRetention` vars be changed to something like these?

Putting the system they apply to at the start so it is a bit easier to identify from the start, rather than the middle.
```
BackupDefaultMonthlyRetention
BackupDefaultWeeklyRetention
BackupDefaultDailyRetention
```

And the feature flag var to match how others are defined with something like this?
```
LFFBackupWeeklyRandom
```

Reply:
@cdchris12 I added this jus after you updated the code, so not sure if you saw it

Reply:
Yep, I got an email. Working on it now. :)